File: benchmarks/maps/bench.dart
int map_test() {
  var map = <int, int>{};
  for (int i = 0; i < 10000; i++) {
    map[i] = i * 2;
  }
  int sum = 0;
  for (int i = 0; i < 10000; i++) {
    sum += map[i]!;
  }
  return sum;
}
void main() {
  print(map_test());
}

File: benchmarks/maps/bench.rs
use std::collections::HashMap;
fn map_test() -> i64 {
    let mut map = HashMap::new();
    let mut i = 0;
    while i < 10000 {
        map.insert(i, i * 2);
        i += 1;
    }
    
    let mut sum = 0;
    i = 0;
    while i < 10000 {
        sum += map.get(&i).unwrap();
        i += 1;
    }
    sum
}
fn main() {
    println!("{}", map_test());
}
